assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -161,6 +161,11 @@ che.docker.always_pull_image=true
 # to be able to launch their own runtimes which are embedded Docker containers.
 che.docker.privileged=false
 
+# This parameter allows to specify custom security options for the created docker container
+# seccomp:unconfined is the default for kubernetes, but not for docker. This is needed
+# for debugging with gdbserver
+che.docker.securityopt=seccomp:unconfined
```

Comment:
Can you elaborate on how to set this option to default docker behavior?

Reply:
@garagatyi default would be an empty parameter